ros2param/package.xml

```diff
@@ -20,6 +20,10 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_xmllint</test_depend>
+  <test_depend>launch</test_depend>
+  <test_depend>launch_ros</test_depend>
+  <test_depend>launch_testing</test_depend>
+  <test_depend>launch_testing_ros</test_depend>
```

Comment:
Even though it doesn't seem like we need to depend on `launch_testing_ros` (e.g. it is not explicitly used anywhere), if I remove this dependency, then I get the error:

```
TypeError: setUpClass() missing 4 required positional arguments: 'launch_service', 'proc_info', 'proc_output', and 'rmw_implementation'
```

@hidmic Any idea why this is the case?

Reply:
With e166f03, we need the test dependency on `launch_testing_ros` anyways. But it still would
be good to know the answer to the original issue for posterity.